lib/fake_idp/application.rb

```diff
@@ -1,5 +1,3 @@
-require_relative "encryptor"
```

Comment:
Why don't we need to include this? Are we not using it? If not, why did we add the class?